src/components/filterItem.tsx

```diff
@@ -20,7 +20,7 @@ function FilterItem({ label, data, icon }: FilterItemProps) {
           icon={icon}
           className="text-xs font-extrabold text-info"
         />
-        <p className="text-bold text-info">{label}</p>
+        <p className="text-bold text-info">{label.charAt(0).toUpperCase() + label.slice(1)}</p>
```

Comment:
hint: This logic could have been moved to a method with a name that is easy to understand.

Reply:
Done.